easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.0.6-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'BerkeleyGW'
+version = "1.0.6"
+
+homepage = 'http://www.berkeleygw.org'
+description = """The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
+                 properties and the optical responses of a large variety of materials from bulk
+                 periodic crystals to nanostructures such as slabs, wires and molecules."""
```

Comment:
no need for the long indent, this will probably make the output of `module help BerkeleyGW` looks rather weird...

just use a single space to indent the continuation lines